dataikuapi/dssclient.py

```diff
@@ -329,17 +329,21 @@ def sql_query(self, query, connection=None, database=None, dataset_full_name=Non
     # Users
     ########################################################
 
-    def list_users(self):
+    def list_users(self, as_objects=False):
```

Comment:
Could be nice to somehow pass the GET parameter lastActivity and retrieve the activity of the users at the same time